### PR TITLE
in_elasticsearch: Add a documentation

### PR DESCRIPTION
**Pull request by cosmo0920**
Corresponding PR is: https://github.com/fluent/fluent-bit/pull/6685

Signed-off-by: Hiroshi Hatake <hatake@calyptia.com>

### CODE REVIEW

File: pipeline/inputs/elasticsearch.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Elasticsearch (Bulk API)
+
+The **elasticsearch** input plugin, allows to handle Elasticsearch and also OpenSearch Bulk API requests.
```
Comment (patrick-stephens) [inline pipeline/inputs/elasticsearch.md:3]:
```suggestion
The **elasticsearch** input plugin, allows the handling of Elasticsearch and also OpenSearch Bulk API requests.
```

File: pipeline/inputs/elasticsearch.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+| meta\_key | Specify a key name for meta information. |
+| hostname | Specify hostname or FQDN. This parameter is effective for sniffering node information. |
+
+## Getting Started
+
+In order to start performing the checks, you can run the plugin from the command line or through the configuration file:
+
+### Command Line
+
+From the command line you can let Fluent Bit handles Bulk API requests with the following options:
```
Comment (patrick-stephens) [inline pipeline/inputs/elasticsearch.md:23]:
```suggestion
From the command line you can configure Fluent Bit to handle Bulk API requests with the following options:
```

File: pipeline/inputs/elasticsearch.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    buffer_chunk_size 5M
+
+[OUTPUT]
+    name stdout
+    match *
+```
+
+#### Ingesting from beats series
+
+Ingesting from beats series agents is also supported.
```
Comment (patrick-stephens) [inline pipeline/inputs/elasticsearch.md:62]: Probably link out to beats series agents.

**Review by patrick-stephens — CHANGES_REQUESTED** [review]
Let's try to use `Fluent Bit` everywhere rather than any other alternative `fluent-bit`, `fluent bit` or `fluent bit`.

File: pipeline/inputs/elasticsearch.md (hunk 3 above)
Comment (cosmo0920) [reply pipeline/inputs/elasticsearch.md:62]: Good point. I added their links in https://github.com/fluent/fluent-bit-docs/pull/1027/commits/c3ce2b8325ce676c0392272124c15b049cf10274.

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**